index.js

```diff
@@ -432,7 +434,23 @@ module.exports = {
 
       return routes;
     },
+    /**
+     * gets either the commonPathItemObjectResponses from settings with possibly merged responses if the flag in the settings is false and gets exclusively your
+     * customCommonPathItemResponses from openapi service settings if the flag in settings has been overwritten with true
+     * 
+     * @returns commonPathItemObjectResponses: either the commonPathItemObjectResponses from settings with possibly merged responses or exclusively your custom commonPathItemObjectResponses
+     */
+    getCommonPathItemObjectResponses(){
+      //takes exclusively customCommonPathItemObjectResponses from the openapi service settings if the useCustomCommonPathItemObjectResponses flag in the settings has been overwritten
+      //to true by openapi service settings
+      if(this.settings.useCustomCommonPathItemObjectResponses) {
+        return this.settings.customCommonPathItemObjectResponses
+      }
+      //returns the default commonPathItemObjectResponses from settings with possibly merged responses from openapi service settings
+      return this.settings.commonPathItemObjectResponses
```

Comment:
@hkess mmm... i'm confused. 
What's another boolean field for?
What's the other customCommonPathItemObjectResponses parameter for?

You can override commonPathItemObjectResponses in settings. Is equal to useCustomCommonPathItemObjectResponses=true and customCommonPathItemObjectResponses = {bla-bla}

Reply:
Currently there is no way of removing the module's common responses, you can only overwrite them. Moleculer uses a deep-merge to join the settings of the service and the module's settings. This however means you will always have a 200, 401, 422, default response even if you only want a 200 response.


In order to solve this, I added a flag (`useCustomCommonPathItemObjectResponses`) to use an empty object (`customCommonPathItemObjectResponses`) as default for the common responses. This object can be used to merge only your custom responses without using the module's default.

Reply:
@hkess ok. I think in this case we should make the getCustomCommonPathItemObjectResponses() method instead of customCommonPathItemObjectResponses object to solve this problem. 3 extra attributes are too much(customCommonPathItemObjectResponses,useCustomCommonPathItemObjectResponses,customCommonPathItemObjectResponses) and will confuse people.

Reply:
Writing a `getCustomCommonPathItemObjectResponses()` method in order the both attributes will not work, because the user still needs to supply his own `commonPathItemObjectResponses` somewhere in his service. Doing that in `commonPathItemObjectResponses` will not work, because it will merge with the modules `commonPathItemObjectResponses` responses. There are 2 valid ways now to supply your responses (either using the empty module object `customCommonPathItemObjectResponses` to avoid using the modules responses or the filled module object `commonPathItemObjectResponses` in order to use the responses from the module). Therefore you would need a way to know which approach to take and thats why the flag is necessary.

I see 2 solutions for removing the need of the aditional properties:

1. empty `commonPathItemObjectResponses` in Module:
we could remove the modules 200, 401, 422, default responses in `commonPathItemObjectResponses`, which means only the `commonPathItemObject` responses you pass in your service will shown.

2. no option to not have 200, 401, 422, default responses:
we dont remove the modules 200, 401, 422, default responses in `commonPathItemObjectResponses`, which means there will always be responses for 200, 401, 422, default even if you dont want all of them.

Reply:
@hkess user can put own commonPathItemObjectResponses in overided getCustomCommonPathItemObjectResponses() method. ex:
```
getCustomCommonPathItemObjectResponses() {
  return {
      200: {
        $ref: "#/components/responses/ReturnedData",
      },
      401: {
        $ref: "#/components/responses/UnauthorizedError",
      },
      422: {
        $ref: "#/components/responses/ValidationError",
      },
      default: {
        $ref: "#/components/responses/ServerError",
      },
    }
}
```

Reply:
okay understood, i removed all previous changes and added getCustomCommonPathItemObjectResponses() so the user is able to overwrite it in his service.
```
    /**
     * gets commonPathItemObjectResponses, if you want to use your own responses, 
     * overwrite this method in your service and return your own responses.
     * 
     * @returns commonPathItemObjectResponses the responses object from path item objects
     */
    getCommonPathObjectResponses(){
      return this.settings.commonPathItemObjectResponses
    }
```